### PR TITLE
WooCommerce price suffix

### PR DESCRIPTION
**Pull request by cliffordp**
5min demo: https://share.getcloudapp.com/04ugQn8e

[ETP-33] 

[ETP-33]: https://moderntribe.atlassian.net/browse/ETP-33

### CODE REVIEW

File: src/Tribe/Commerce/Currency.php [hunk 1]
```diff
@@ -838,19 +839,43 @@ public function get_formatted_currency_with_symbol( $amount, $post_id, $provider
...
+	 * @return string
+	 */
+	private function get_amount_suffix_html( $amount, $provider = null ) {
+		$suffix = '';
+
+		if (
+			'Tribe__Tickets_Plus__Commerce__WooCommerce__Main' === $provider
+			&& method_exists( 'WC_Product', 'get_price_suffix' )
+		) {
+			$suffix = ( new WC_Product() )->get_price_suffix( $amount );
```
Comment (sc0ttkclark) [inline src/Tribe/Commerce/Currency.php]: Shorthand is great, readable is better

```suggestion
			$product = new WC_Product();
			
			$suffix = $product->get_price_suffix( $amount );
```

File: src/Tribe/Commerce/Currency.php [hunk 2]
```diff
@@ -838,19 +839,45 @@ public function get_formatted_currency_with_symbol( $amount, $post_id, $provider
...
+	 * Get the amount suffix, such as for WooCommerce's taxation suffix.
+	 *
+	 * @since TBD
+	 *
+	 * @param int|string  $amount
+	 * @param string|null $provider
+	 *
+	 * @return string
+	 */
+	private function get_amount_suffix_html( $amount, $provider = null ) {
```
Comment (sc0ttkclark) [inline src/Tribe/Commerce/Currency.php]:
```suggestion
	public function get_amount_suffix_html( $amount, $provider = null ) {
```

File: src/Tribe/Commerce/Currency.php [hunk 3]
```diff
@@ -838,19 +839,45 @@ public function get_formatted_currency_with_symbol( $amount, $post_id, $provider
...
 
+	/**
+	 * Get the amount suffix, such as for WooCommerce's taxation suffix.
+	 *
+	 * @since TBD
+	 *
+	 * @param int|string  $amount
+	 * @param string|null $provider
+	 *
+	 * @return string
```
Comment (sc0ttkclark) [inline src/Tribe/Commerce/Currency.php]: param and return need comments

File: readme.txt [hunk 4]
```diff
@@ -119,6 +119,7 @@ Currently, the following add-ons are available for Event Tickets:
 
 = [4.11.2] TBD =
 
+* Fix - Display WooCommerce price taxation suffix, if applicable [ETP-33]
```
Comment (sc0ttkclark) [inline readme.txt]:
```suggestion
* Fix - Display WooCommerce price taxation suffix, if applicable. [ETP-33]
```

File: src/Tribe/Commerce/Currency.php (hunk 3 above)
Comment (cliffordp) [reply src/Tribe/Commerce/Currency.php]: 👍 done

File: src/Tribe/Commerce/Currency.php [hunk 5]
```diff
@@ -838,19 +839,48 @@ public function get_formatted_currency_with_symbol( $amount, $post_id, $provider
...
+	 *
+	 * Regardless of passed $amount, depending on commerce settings, could return a suffix of
+	 * `<small class="woocommerce-price-suffix">excl. VAT</small>`
+	 *
+	 * @since TBD
+	 *
+	 * @param int|string  $amount   Price amount for which to get the formatted result.
+	 * @param string|null $provider Ticket provider class name. Currently defaults to and only supports Woo's.
+	 *
+	 * @return string
```
Comment (sc0ttkclark) [inline src/Tribe/Commerce/Currency.php]:
```suggestion
	 * @return string Amount suffix, if any is given by the Commerce provider.
```

**Pull request merged**